pyproject.toml

```
@@ -0,0 +1,70 @@
+# Copyright 2024 LINE Corporation
+#
+# LINE Corporation licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+[build-system]
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "centraldogma-python"
+dynamic = ["version"]
+description = "Python client library for Central Dogma"
+readme = "README.md"
+urls = {Homepage = "https://github.com/line/centraldogma-python"}
+authors= [{name = "Central Dogma Team", email = "dl_centraldogma@linecorp.com"}]
+license = { text = "Apache License 2.0" }
```

Since we already have a license file, `file` seems better instead of `text`. https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license
```suggestion
license = {file = "LICENSE"}
```